.php-cs-fixer.dist.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+$finder = (new PhpCsFixer\Finder())
+    ->exclude(__DIR__ . '/vendor')
+    ->in(__DIR__ . '/src')
+    ->name('*.php');
+
+return (new PhpCsFixer\Config())
+    ->setUsingCache(false)
+    ->setRiskyAllowed(true)
+    ->setRules([
```

Comment:
add [trailing comma rule](https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/blob/master/doc/rules/control_structure/trailing_comma_in_multiline.rst) with elements subset: `['arguments', 'arrays', 'match', 'parameters']` and run it to fix some inconsistencies.